$OpenBSD: patch-configure_in,v 1.2 2018/04/28 16:01:52 jasper Exp $

- fix linking with lld by explicitly requesting gthread-2.0
- clang doesn't pass -export-dynamic to ld(1), use -rdynamic to unbreak
  libglade.

Index: configure.in
--- configure.in.orig
+++ configure.in
@@ -13,7 +13,7 @@ AC_PROG_CC
 AC_PROG_INSTALL
 
 # Checks for libraries.
-PKG_CHECK_MODULES(GNAUGHTY, gtk+-2.0 >= 2.0.0 gconf-2.0 libglade-2.0)
+PKG_CHECK_MODULES(GNAUGHTY, gtk+-2.0 >= 2.0.0 gthread-2.0 gconf-2.0 libglade-2.0)
 #GNAUGHTY_LIBS="${GNAUGHTY_LIBS}"
 
 AM_GCONF_SOURCE_2
@@ -31,7 +31,8 @@ AC_SUBST(PCRE_LIBS)
 AC_SUBST(GNAUGHTY_LIBS)
 AC_SUBST(GNAUGHTY_CFLAGS)
 
-CFLAGS="$CFLAGS -Wall -export-dynamic"
+CFLAGS="$CFLAGS -Wall"
+LDFLAGS="$LDFLAGS -rdynamic"
 
 # Checks for header files.
 AC_HEADER_STDC
